### PR TITLE
Add question 4 and subsection

### PR DESCRIPTION
**Pull request by bilbof**
This adds pages asking people to say if they can offer transport, and asking what kind of transport they can offer.

Screenshot 2020-03-21 at 10 50 59

Screenshot 2020-03-21 at 10 50 54

https://trello.com/c/b0M4OBsx/25-implement-question-4-offer-transport
https://trello.com/c/SMUmY3ym/26-implement-question-41

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -6,3 +6,7 @@ inherit_gem:
 inherit_mode:
   merge:
     - Exclude
+
+Metrics/BlockLength:
```
Comment (huwd) [inline .rubocop.yml]: Nice spot!

File: app/views/coronavirus_form/offer_food.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 <%= render "coronavirus_form/validation_error" %>
 <%= form_tag({},
-  "data-module": "track-coronavirus-form-business-hotel-rooms",
+  "data-module": "track-coronavirus-form-business-offer-food",
```
Comment (huwd) [inline app/views/coronavirus_form/offer_food.html.erb:12]: Question, do we even need this line if we shouldn't have analyitics?

File: app/views/coronavirus_form/offer_transport.html.erb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  <meta name="description" content="<%= t('coronavirus_form.offer_transport.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= render "coronavirus_form/validation_error" %>
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-business-offer-transport",
```
Comment (huwd) [inline app/views/coronavirus_form/offer_transport.html.erb:12]: Question, do we even need this line if we shouldn't have analyitics?

File: app/views/coronavirus_form/transport_type.html.erb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  <meta name="description" content="<%= t('coronavirus_form.transport_type.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= render "coronavirus_form/validation_error" %>
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-business-transport-type",
```
Comment (huwd) [inline app/views/coronavirus_form/transport_type.html.erb:12]: Question, do we even need this line if we shouldn't have analyitics?

File: spec/controllers/coronavirus-form/offer_transport_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+  describe "GET show" do
+    it "renders the form" do
+      get :show
+      expect(response).to render_template(current_template)
+    end
+  end
+
+  describe "POST submit" do
+    let(:selected_yes) { I18n.t("coronavirus_form.offer_transport.options.option_yes.label") }
```
Comment (huwd) [inline spec/controllers/coronavirus-form/offer_transport_controller_spec.rb:17]: much better than the way i had it, will steal this for my branch

File: app/views/coronavirus_form/offer_transport.html.erb (hunk 3 above)
Comment (bilbof) [reply app/views/coronavirus_form/offer_transport.html.erb:12]: I've kept it in as we have it for the others, since we may need analytics.

**Review by huwd — APPROVED** [review]

**Pull request merged**